### ISSUE TITLE
InvalidRepositoryException github.com/DmitryBaranovskiy/eve.git

### ISSUE THREAD

**Issue by vladyslavstartsev** [mdmsoft/yii2-admin#353]
Hi! I'm developing legacy yii2 project (https://github.com/adam2314/linet3) that uses your dependency (see [this](https://github.com/adam2314/linet3/blob/master/protected/composer.json)) when I've installed the app  (I mean `git clone` and the `composer update`) it caused the `InvalidRepositoryException github.com/DmitryBaranovskiy/eve.git`, I don't know how is that possible, but I got it. 
I found it out, this way - first I've tried to install all of the dependencies and then after I got error, I started to delete dependencies line by line and got errors, but when I've deleted yours I got fully working app

**Comment by makaronnik** [comment]
Faced with the same problem. There is a solution?

**Comment by vladyslavstartsev** [comment]
@makaronnik yes, I've set my `composer.json` to have special commit like this

```json
    "require": {
        "mdmsoft/yii2-admin" : "dev-master#1c0887eff2fe5eac924c413219d20e761d5e1d9a"
    }
```

**Comment by mdmunir** [comment]
```
"require": {
        "mdmsoft/yii2-admin" : "~2.0"
    }
```

**Comment by vladyslavstartsev** [comment]
@mdmunir this doesn't work (at least when I've check), it give the error that I have described

**Comment by schmunk42** [comment]
Related:
- https://github.com/dmstr/yii2-adminlte-asset/issues/133#issuecomment-337179853
